src/main/java/com/gamegoo/domain/champion/MemberChampion.java

```diff
@@ -32,8 +33,10 @@ public void setMember(Member member) {
             this.member.getMemberChampionList().remove(this);
         }
         this.member = member;
+        if (this.member.getMemberChampionList() == null) {
```

Comment:
혹시 여기에서 이 조건을 추가하신 이유가 있으신가요?
`    @OneToMany(mappedBy = "member", cascade = CascadeType.ALL)
    private List<MemberChampion> memberChampionList = new ArrayList<>();`
    member 엔티티에서 이렇게 초기화를 해 주어서 저는 memberChampionList가 null인 경우가 없을 거라고 생각했었는데 혹시 이 조건+코드가 없으면 뭔가 오류가 나는건지 궁금합니다..!

Reply:
네ㅠㅠ 이상하게 @OneToMany를 했음에도 불구하고 이런 에러가 발생하더라구요..
Cannot invoke \"java.util.List.add(Object)\" because the return value of \"com.gamegoo.domain.Member.getMemberChampionList()\" is null
그래서 추가했습니다!

Reply:
헉 그렇군요...... ㅠㅠㅠㅠ 왜 그런건지 제가 나중에 따로 찾아보겠습니다.. 그럼 이대로 머지해주시면 될 것 같아요 !!

Reply:
넵 감사합니다!